src/main/java/com/karankumar/bookproject/backend/entity/BookFormat.java

```diff
@@ -0,0 +1,37 @@
+/*
+    The book project lets a user keep track of different books they would like to read, are currently
+    reading, have read or did not finish.
+    Copyright (C) 2020  Karan Kumar
+
+    This program is free software: you can redistribute it and/or modify it under the terms of the
+    GNU General Public License as published by the Free Software Foundation, either version 3 of the
+    License, or (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful, but WITHOUT ANY
+    WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+    PURPOSE.  See the GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along with this program.
+    If not, see <https://www.gnu.org/licenses/>.
+ */
+
+
+package com.karankumar.bookproject.backend.entity;
+
+public enum BookFormat {
+    // This should be kept in alphabetical order
+    HARDBACK("Hardback"),
+    PAPERBACK("Paperback"),
+    EBOOK("eBook");
```

Comment:
Could you please ensure this is in alphabetical order? It's not an issue now, but if we add more later, it'll help